cmd/deploy/deploy.go

```diff
@@ -278,6 +281,27 @@ func (dc *deployCommand) runDeploy(ctx context.Context, cwd string, opts *Option
 	oktetoLog.Debugf("starting server on %d", dc.proxy.GetPort())
 	dc.proxy.Start()
 
+	oktetoLog.LogIntoBuffer("Deploying app '%s'...", opts.Name)
```

Comment:
Let's not mention `app` in any place. We don't have the concept of `app` in the Okteto UI.
I would just say `"Deploying '%s'...", opts.Name`